spec/controllers/admin/subscriptions_controller_spec.rb

```diff
@@ -754,7 +754,7 @@
     end
 
     it "assigns data to instance variables" do
-      controller.send(:load_form_data)
+      controller.__send__(:load_form_data)
```

Comment:
Just my two cents here:
- I believe the use of `__send__` method should be avoided as it breaks the encapsulation principle of OOP. Private methods are meant to be used within the classes they are defined in, and this method just invokes the methods that were not intended to be used outside of the class.
- I prefer to use its public counterpart (`public_send`) where there is a need to call the methods dynamically.
- If we know which method to call, then we should directly call it, rather than using another method to do so.
- This PR can be the best opportunity to identify these private method calls and make them either public or see a better alternative.
- What's your take on this, @openfoodfoundation/core-devs?

Reply:
Hey - thanks for the review. There were similar thoughts which occurred to me as well - I had a quick look at a few of the methods in question the ones I looked at seemed to be relatively short and straightforward (but also made sense as private methods eg. `load_form_data` which just sets a few instance variables for the controller class).
But really it made me wonder if these private methods need to be tested at all - as long as the public methods that call them are tested - but there's also an argument that it could make those tests more complex too. I'd be curious to hear others' opinions as well!

Reply:
I agree with what both of you are saying. We should really test the logic of these private methods via the public interface of the object. And I believe, it can reveal design issue in the code. We should remove these `send` but this not really the scope of this PR.